.gitignore

```
@@ -0,0 +1 @@
+/out/production/sample1/com/company/ErrorMessage.class
```

```suggestion
/out/
```

Ignore the entire folder

---

changed as suggested